resources/sql/application-queries.sql

```diff
@@ -1,3 +1,11 @@
--- name: add-application-query<!
+-- name: yesql-add-application-query<!
```

Comment:
Should be use this yesql-prefix elsewhere now too to avoid having two different styles for query names?